src/things/mod.rs

```diff
@@ -90,19 +90,19 @@ impl FromStr for ThingType {
 }
 
 #[derive(Debug, Clone)]
-pub struct SubredditSet(HashSet<String>);
+pub struct ShredditSet(HashSet<String>);
```

Comment:
fun idea to rename it this, but i'd prefer keeping things super simple / boring so it's clear that these are subreddits =) could you rename to `Subreddits`?

Reply:
ah, i see - it's used for comment / post IDs too

Reply:
Added typealiases for them, what do you think about this solution?

Reply:
i like that approach!